packages/plugins/integration-zapier/src/lib/zapier-auth-code.service.ts

```
@@ -0,0 +1,241 @@
+            return true; // Default to single-instance if not defined
+        }
+
+        if (instanceCount.toLowerCase() === 'true') {
+            return true; // Handle boolean "true"
+        }
+
+        if (instanceCount.toLowerCase() === 'false') {
+            return false; // Handle boolean "false"
+        }
+
+        const parsedCount = parseInt(instanceCount, 10);
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)